node/cli/src/chain_spec/bootstrap.rs

```diff
@@ -116,11 +121,13 @@ fn cord_custom_config_genesis(config: ChainParams) -> serde_json::Value {
 		initial_council_members,
 		initial_tech_committee_members,
 		initial_sudo_key,
+		prefix,
```

Comment:
Have to set `prefix` as `network_id` which in-turn gets passed to create `serde_json`.
Or use a variable for consistency `prefix = network_id.clone()`